torch/_inductor/codegen/wrapper.py

```diff
@@ -203,9 +204,9 @@ def __str__(self):
 class MemoryPlanningState:
     def __init__(self):
         super().__init__()
-        self.reuse_pool: Dict[
-            ReuseKey, List[FreeIfNotReusedLine]
-        ] = collections.defaultdict(list)
+        self.reuse_pool: Dict[ReuseKey, List[FreeIfNotReusedLine]] = (
+            collections.defaultdict(list)
+        )
```

Comment:
This change is causing lints to fail:
https://hud.pytorch.org/pr/pytorch/pytorch/122632#23375048104

Reply:
Emm, this is changed by runnning `lintrunner -f` locally. Let me revert this change

Reply:
Maybe wrong versions.  You can do `make setup_lint`.